samples/bluetooth/observer/overlay_bbc_microbit-bt_ll_sw_split.conf

```diff
@@ -0,0 +1,59 @@
+# Adjust Stack Sizes for reduce RAM usage
+CONFIG_MAIN_STACK_SIZE=1024
+CONFIG_IDLE_STACK_SIZE=128
+CONFIG_ISR_STACK_SIZE=1024
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=1024
+
+# Enable Extended Scanning
+# CONFIG_BT_EXT_ADV=y
+
+# Set maximum scan data length for Extended Scanning
+CONFIG_BT_EXT_SCAN_BUF_SIZE=192
```

Comment:
The config enabling extended scanning is commented out, but below the buf size set. Uncomment?